src/main.cc

```diff
@@ -798,6 +790,7 @@ shut_down(int sig)
 void
 sig_child(int sig)
 {
+    (void)sig;
```

Comment:
Another case where the new force-use code would disable a useful warning when/if `sig` becomes unused in the code that currently uses it. Please move to the refactored `#if` statement:

```C++
#if !defined(_SQUID_WINDOWS_) && !defined(HAVE_SIGACTION)
    signal(sig, sig_child);
#else
    (void)sig;
#endif

Reply:
Sure, better readability

Reply:
done in commits 3bc5b438f and 87b68fe95